src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+$primary_color: #ddd;
+$secondary_color: #fff;
+$btn_border_color: red;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'montserrat', sans-serif;
+  background: transparent;
+  .slider {
+    display: none;
+  }
+}
+
+.bgimg {
+  background: url('../css/img/cv.jpg');
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  filter: blur(5px);
+}
+
+.header {
+  max-width: 1000px;
+  width: 100%;
+  padding: 0 auto;
```

Comment:
`auto` is an invalid padding value.
![Screen Shot 2019-09-12 at 9 26 46 AM](https://user-images.githubusercontent.com/123074/64736768-86a30f80-d53f-11e9-8214-05df08e6b9e4.png)